code/assurance/src/main/java/com/adobe/marketing/mobile/assurance/AssuranceAppState.kt

```diff
@@ -68,13 +68,28 @@ internal class AssuranceAppState {
         object Connected : SessionPhase()
     }
 
+    /**
+     * Represent a log line in the Assurance Status UI.
+     * @param level the visibility of the log line
+     * @param message the message to display
+     */
+    internal data class StatusLog(val level: AssuranceConstants.UILogColorVisibility, val message: String)
+
     private val _sessionPhase = mutableStateOf<SessionPhase>(SessionPhase.Disconnected())
 
     /**
      * Represents the current phase of the assurance session.
      */
     val sessionPhase: State<SessionPhase> = _sessionPhase
 
+    private val _statusLogs = mutableStateOf<List<StatusLog>>(emptyList())
```

Comment:
nit: can we consider using a more succinct expression -> `private val _statusLogs = mutableStateOf(emptyList<StatusLog>())`?